examples/example-app-router-migration/package.json

```
@@ -10,7 +10,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "^14.0.3",
+    "next": "^14.0.4",
```

Wouldn't this be better for examples?

```suggestion
    "next": "latest",
```
It's used in next.js examples

https://github.com/vercel/next.js/blob/c4adae89b17a6f64cc0abe8e697a88b9e1df1814/examples/api-routes/package.json#L9

---

Hi Ahmed! I think it's a good idea to pin the major version of Next.js so we can upgrade in a controlled fashion in case breaking changes are introduced.